src/main.ts

```diff
@@ -35,9 +36,28 @@ async function run(): Promise<void> {
   }
   core.endGroup()
 
+  if (config.postprocess) {
+    core.startGroup('Postprocess')
+    core.debug(`Invoking ${config.postprocess} with ${filename}...`)
+    try {
+      const raw = execSync(
+        `deno run -q -A ${config.postprocess} ${filename}`
```

Comment:
Would there be any reason to limit the permissions here?

Reply:
I mean, I think it's probably sensible to allow people to control it, maybe with a separate setting like `postprocess_deno_permissions` that defaults to `-A` so people don't have to set it. But for now, let's just ship this.